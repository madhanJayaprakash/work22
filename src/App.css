.navbar-toggle.pull-left {
  margin-left: 15px;
}
@media (min-width: 768px) {
  .pull-sm-left {
    float: left !important;
  }
}

hr.solid {
  border-top: 1.3px solid rgb(218, 215, 215);
}

hr.blogcls{
  border-top: 1px solid rgb(218, 215, 215);
}
.authorShow{
  display: block;
}
.authorHide{
  display: none !important;
}
.articlecardcls{
  cursor: pointer;
}
.readmorecls{
  text-decoration: none;
}