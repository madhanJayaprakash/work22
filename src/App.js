import React from 'react';
import Container from 'react-bootstrap/Container';
import Header from './components/Header';
import Article from './components/Article';
import './App.css';

function App() {
  
  return (
    <div className="App">
      <Container fluid className='gx-0'>
        <Header />
        <Article/>
      </Container>
    </div>
  );
}

export default App;
