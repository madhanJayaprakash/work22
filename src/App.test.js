import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import App from './App'

Enzyme.configure({ adapter: new Adapter() })

describe('Test Case For App', () => {
 
  it('should render button', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot();
  });


})

