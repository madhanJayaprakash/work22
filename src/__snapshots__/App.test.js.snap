// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Case For App should render button 1`] = `ShallowWrapper {}`;
